docs/guides-blog.md

```diff
@@ -42,7 +42,7 @@ The only required field is `title`; however, we provide options to add author in
 
 - `author` - The text label of the author byline.
 - `authorURL` - The url associated with the author. This could be a Twitter, GitHub, Facebook account, etc.
-- `authorFBID` - The Facebook ID that is used to extract the profile picture.
+- `authorFBID` - The Facebook ID that is used to extract the profile picture. (Note: If you prefer to use a different image than the Facebook profile picture, put `authorImageURL` instead.)
```

Comment:
`authorImageURL` should be documented as a separate field

Reply:
I updated the documentation. Let me know if you want any edits, was tricky to know the best way to write this one clearly.